package.json

```diff
@@ -102,7 +102,9 @@
     "ansi-styles": "^3.2.1",
     "antd-theme-generator": "^1.1.4",
     "antd-tools": "^6.4.4",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.4.2",
```

Comment:
Not need to change `dependencies`

Reply:
Getting this when removing them:
```
Cannot find module 'babel-core'
```

Reply:
feature branch works well

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/507615/50468040-d3aa7a00-09e0-11e9-86cb-b5a8aef15ea0.png">